config/ambd/.gn

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+
+# The location of the build configuration file.
+buildconfig = "${build_root}/config/BUILDCONFIG.gn"
+
+# CHIP uses angle bracket includes.
+check_system_includes = true
+
+default_args = {
+  target_cpu = "ambd"
```

Comment:
nit: is this name we chose or is it mandated by some gcc or other flags?

I am wondering if we could have a longer "ameba-d" or "amebad" for easier parsing.

Reply:
Thanks for the feedback,
The 'target_cpu' argument will be changed to 'ameba' instead.
The platform name will be changed to the camel style as 'Ameba' instead.

Reply:
Is this actually a CPU?

I looked at the website and it says it's an ARM Cortex M4?

Reply:
Unresolving based on comment from Michael - should target_cpu be a 'CPU'? are there other things that can be used to identify the platform instead?

Reply:
Thanks for feedback. I changed target_cpu to "arm".